### DISCUSSION TITLE
When is the Windows version coming?

### DISCUSSION THREAD

**Discussion by sbc640964** [beyondcode/herd-community#469] [General]
I bought a new computer?
Do I have a reason to wait?

**Comment by sbc640964** [comment]
?

**Comment by sbc640964** [comment]
@mpociot, @sschlein I would love to get an answer, should I install another development environment?

**Comment by JaberWiki** [comment]
> I bought a new computer? Do I have a reason to wait?

Already released for Windows.
[https://herd.laravel.com/windows](https://herd.laravel.com/windows)
